styles/addboard.module.css

```
@@ -0,0 +1,145 @@
+.top-wrapper {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+.top-wrapper-board {
+  font-size: 20px;
+  font-weight: 700;
+  line-height: 23.87px;
+  color: #1f2937;
+}
+.register-button {
+  height: 42px;
+  border-radius: 8px;
+  border: none;
+  padding: 12px 23px;
+  background-color: #3692ff;
+  color: #fff;
+  font-size: 16px;
+  font-weight: 600;
+  line-height: 19.09px;
+}
```

가독성과 유지보수를 위해 각 블럭 사이에 엔터값을 넣어주시면 더 좋을 것 같습니다.
모든 요소가 붙어 있으면 추가, 삭제 작업 시 의도치 않게 다른 요소의 코드 블럭을 건드릴수도 있어서요!

```suggestion
.top-wrapper {
  display: flex;
  justify-content: space-between;
  align-items: center;
}

.top-wrapper-board {
  font-size: 20px;
  font-weight: 700;
  line-height: 23.87px;
  color: #1f2937;
}

.register-button {
  height: 42px;
  border-radius: 8px;
  border: none;
  padding: 12px 23px;
  background-color: #3692ff;
  color: #fff;
  font-size: 16px;
  font-weight: 600;
  line-height: 19.09px;
}
```